src/huggingface_hub/hf_file_system.py

```diff
@@ -557,7 +557,7 @@ def info(self, path: str, refresh: bool = False, revision: Optional[str] = None,
     def exists(self, path, **kwargs):
         """Is there a file at the given path"""
         try:
-            self.info(path, expand_info=False, **kwargs)
+            self.info(path, **{**kwargs, "expand_info": False})
```

Comment:
this was causing a `TypeError: huggingface_hub.hf_file_system.HfFileSystem.info() got multiple values for keyword argument 'expand_info'`